ansible_base/authentication/views/authenticator.py

```diff
@@ -21,6 +21,12 @@ class AuthenticatorViewSet(AnsibleBaseDjangoAppApiView, ModelViewSet):
 
     def destroy(self, request, *args, **kwargs):
         instance = self.get_object()
+
+        # check if there is at least one other enabled authenticator...
+        if not Authenticator.objects.filter(enabled=True).exclude(id=instance.id).exists():
```

Comment:
I would prepend to this `if` condition, `instance.enabled and`. If somehow all enabled authenticators already got deleted, I don't want to block users doing cleanup of data which doesn't affect that situation.